Electric-Phantoms/welcome-to-nashville
# Adding some json

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

git fetch --all
git checkout adding-some-json
run http and json servers
open localhost 8080
do a search for each type and make a save into the itinerary
clear each of the saved results in the itinerary and make sure there are no problems



# Checklist:
